downstream/modules/platform/con-ocp-supported-install.adoc

```diff
@@ -32,6 +32,15 @@ If existing components have already been deployed, you must specify these compon
 * Same with {ControllerName} and {EDAName}
 |===
 
+== Modifying the number of simultaneous rulebook activations during or after {EDAcontroller} installation
+
+* If you plan to install {EDAcontroller} and modify the number of simultaneous rulebook activations, you must add the `EDA_MAX_RUNNING_ACTIVATIONS` parameter. See the example in appendix link:{URLOperatorInstallation}#eda_max_running_activations[EDA_MAX_RUNNING_ACTIVATIONS].
+
+[NOTE]
+====
+`EDA_MAX_RUNNING_ACTIVATIONS` for {OCPShort} is a global value since there is no concept of worker nodes in {OCPShort}.
+====
+
```

Comment:
@Alex-Izquierdo - here's where I added the note about the global value. Please feel free to tweak this note the way you think it's best and I'll edit it for editorial purposes.

Reply:
If you think it is a better place, LGTM, but the sentence still has an error:
I suggest something like: 
```suggestion
`EDA_MAX_RUNNING_ACTIVATIONS` for {OCPShort} is a global value since there is no concept of worker nodes in {EDAcontroller} for {OCPShort}.
```

There is a concept of worker nodes in OCP, what we don't have is the concept of worker nodes in EDA when it is installed in OCP.

Reply:
Thanks. As stated previously in my comment above, "Please feel free to tweak this note the way you think it's best ..." Thanks for adding edits for technical accuracy.